src/uwtools/apps/srw.py

```diff
@@ -29,15 +30,19 @@ def load_config(self, config_file: str) -> None:
         """
         file_type = get_file_type(config_file)
         if file_type == FORMAT.ini:
-            with open("config.yaml", "w", encoding="utf-8") as file:
+            with open("config.yaml", "w", encoding="utf-8") as f:
                 # Note: This is a temporary path until parsing the SRW directory is implemented.
-                subprocess.run(
-                    f"python config_utils.py -c {config_file} -t $PWD/config_defaults.yaml -o yaml",
-                    capture_output=True,
-                    check=False,
-                    shell=True,
-                    stdout=file,
-                )
+                cmd_components = [
+                    "python config_utils.py",
+                    "-c %s" % config_file,
+                    "-t $PWD/config_defaults.yaml",
+                    "-o yaml",
+                    ">%s" % f,
+                ]
+                cmd = " ".join(cmd_components)
+                result = run(cmd=cmd)
+                if not result.success:
+                    raise UWError(f"Command failed: {cmd}")
```

Comment:
Replace direct `subprocess.run()` call with a call to the new wrapper, which also checks the command's exit status and returns it to Python, so we can respond if the command failed -- in this case by raising an exception.